lib/graphql/language/parser.y

```diff
@@ -372,6 +372,24 @@ def next_token
   end
 end
 
+def get_description(token)
+  comments = []
+
+  loop do
+    token = token.prev_token
+
+    break if token.nil?
+    break if token.name != :COMMENT
+    break if token.next_token.line != token.line + 1
+
+    comments << token.to_s[2..-1] || ""
```

Comment:
This assumes `"# "` for leading a comment, right? Should we support `#` _without_ a space?